js/plugins/firebase/src/index.ts

```diff
@@ -26,8 +26,12 @@ import {
 } from '@genkit-ai/google-cloud';
 export { defineFirestoreRetriever } from './firestoreRetriever.js';
 
+export interface FirebaseTelemetryOptions extends GcpTelemetryConfigOptions {
+  // future: firebase specific telemetry options
+}
+
 export async function enableFirebaseTelemetry(
-  options?: GcpTelemetryConfigOptions
+  options?: FirebaseTelemetryOptions
```

Comment:
I'm worried this is breaking change... how about `GcpTelemetryConfigOptions | FirebaseTelemetryOptions`?

Reply:
all set